dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/SkyminderController.kt

```diff
@@ -0,0 +1,27 @@
+package org.dataland.datalandbackend.controller
+
+import org.dataland.datalandbackend.api.SkyminderAPI
+import org.dataland.skyminderClient.interfaces.DataConnectorInterface
+import org.dataland.skyminderClient.model.ContactInformation
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * Implementation of the API for using Skyminder
+ * @param dataConnector implementation of the DataConnectorInterface that defines how to connect to the data
+ * source (e.g. Skyminder)
+ */
+
+@RestController
+class SkyminderController(@Autowired var dataConnector: DataConnectorInterface) : SkyminderAPI {
+    override fun getDataSkyminderRequest(countryCode: String, companyName: String):
+        ResponseEntity<List<ContactInformation>> {
+        return ResponseEntity.ok(
+            this.dataConnector.getContactInformation(
```

Comment:
Ich finde `dataConnector` und `DataConnector` suboptimale Namen: Das sind doch ganz spezielle Dinge für Skyminder - und nicht irgendwelche generischen Dinge mit Daten. Warum nicht einfach `SkyminderClient` ?

Reply:
Ok. Den Variablennamen könnte man ja zur `skyminderClient` setzen, aber für den Interface-Namen müsste man zuerst im privaten Skyminder-Repo das Interface von `DataConnectorInterface` zu `SkyminderClientInterface` ändern.
Dann neu publishen und hier einfach neu bauen? 
Sollte doch so passen, weil wir hier im Projekt noch im Snapshot Modus sind.

Reply:
Du kannst doch ein neues Release im SkyminderRepo machen? Musst ja keine SnapshotVersionen benutzen.
Ich finde wir sollten in main keine snapshot-versionen benutzen - während der Entwicklung ist das aber natürlich ok.

Reply:
Können wir aber auch nochmal im Team diskutieren - und einen Coding-Guidelines Eintrag zu erstellen.

Reply:
Die Namen enthalten nun explizit skyminder

Reply:
ok